tool/embedded-binaries.ts

```diff
@@ -51,46 +59,73 @@ export async function getEmbeddedProtocol(outPath: string): Promise<void> {
  */
 export async function getDartSassEmbedded(outPath: string): Promise<void> {
   await fs.mkdir(outPath, {recursive: true});
-  let releaseTarball: Buffer;
 
-  console.log('Downloading Dart Sass Embedded.');
+  let latestRelease: {
+    tag_name: string;
+    name: string;
+  };
+
+  // TODO(awjin): Once dart-sass-embedded is no longer under pre-release, call
+  // the Github API instead of manually piecing together the asset URL. The
+  // API exposes only the latest *non*-pre-release versions.
+  // https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-latest-release
+  console.log('Getting Dart Sass Embedded release info.');
   try {
-    // Get the URL of the release asset.
-    //
-    // TODO(awjin): Once dart-sass-embedded is no longer under pre-release, call
-    // the Github API instead of manually piecing together the asset URL. The
-    // API exposes only the latest *non*-pre-release versions.
-    // https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-latest-release
-    let response = await fetch(
-      'https://api.github.com/repos/sass/dart-sass-embedded/releases'
+    const fetchOptions = {
+      redirect: 'follow' as RequestRedirect,
+    } as RequestInit;
```

Comment:
```suggestion
    const fetchOptions: RequestInit = {
      redirect: 'follow' as RequestRedirect,
    };
```

I believe this is a more idomatic way to add a type restriction to a variable (although I wouldn't call myself an expert here).